plugins/chat/assets/javascripts/discourse/connectors/below-footer/chat-message-actions-outlet.js

```diff
@@ -7,10 +7,33 @@ const MSG_ACTIONS_VERTICAL_PADDING = -10;
 export default {
   setupComponent(args, component) {
     const container = getOwner(this);
-    component.chat = container.lookup("service:chat");
+    const chat = container.lookup("service:chat");
+    component.chat = chat;
+    const site = container.lookup("service:site");
+    component.siteService = site;
 
     let popper;
 
+    component.setMessageActionsMobileAnchor = (element, [activeMessage]) => {
```

Comment:
Not sure if this is what you will want for this, I added it back in because otherwise the message actions on mobile has 0 height and width and is offscreen